src/components/Footer/styles.scss

```
@@ -0,0 +1,11 @@
+@import '../../styles/main.scss';
+
+.footer-container {
+  height: rem(32);
```

Should we use `px` unit?
```suggestion
  height: rem(32px);
```

---

I think rem is a unit already, no need to put px scaling it.